coopr-standalone/README.md

```diff
@@ -12,14 +12,13 @@ mvn clean package assembly:single -DskipTests
 ```
 
 ## Install
+
+Unpack Coopr Standalone ZIP onto your system and enter it's directory.
```

Comment:
There's no typo. This is a merge. I'll fix that in the destination branch in another PR.